book/website/ethical-research/law-policy.md

```diff
@@ -0,0 +1,251 @@
+# Ethics and Law, Policy and Human Rights
+
+## Prerequisites / recommended skill level
+None
+
+## Summary
+Law, human rights and policy all relate to ethics, but they are not the same thing.
+
+You should be aware of the laws and policies that apply to you and your work.
+However, this is not sufficient to ensure that your work is ethical. 
+
+Human rights provide a legal framework for protecting individuals. 
+However, it operates at national and international level.
+Ethical work should be grounded in human rights, but sometimes require more guidance to operationalise for particular contexts.
+
+Laws and policies are made by people, and they are subject to negotiations and exercises of power: they can also be changed.
+
+## How this will help you/ why this is useful
+It's useful to know how the ethics that guide your work are underpinned by other frameworks, including law, policy, and human rights. 
+As well as giving you a deeper understanding, it can also help you advocate for ethical work with colleagues and collaborators.
+
+## Law
+
+This is not legal advice. Please read for information only! 
+[Link to central page on 'this book is not a lawyer' - not written yet, but there is an issue open here: https://github.com/alan-turing-institute/the-turing-way/issues/1469)]
+
+### Ethics and law
+
+Laws have been made by humans.
+They reflect the outcomes of negotiations between different people, with different interests and different amounts of power.
+Complying with laws is usually necessary, but *rarely sufficient*, to ensure that you are acting ethically.
+
+### Domestic and regional law
+
+You should be aware of what laws cover your work.
+This will depend on which jurisdictions you work in: usually the geographical area covered by a law.
+You should be aware of the laws that cover you at local, state or regional, national, and international levels. 
+
+You should also be aware what steps you should take to ensure that you are compliant with all relevant laws. 
+This section is not exhaustive!
+It is designed to give you some idea of the kinds of laws you should be aware of.
```

Comment:
I would emphasize that "work" here refers to "data science" or "open data science." If those areas are not exhaustive or incorrect, it would be helpful to specify a bit what type of work the chapter is seeking to address.